maidroid_tool/api.lua

```diff
@@ -52,13 +52,23 @@ function maidroid_tool.register_writer(nodename, options)
 		-- if time is positive, this node is active.
 		if time >= 0 then
 			if time <= duration then
-				meta:set_float("time", time + 1)
+				meta:set_float("time", time + elapsed)
 				meta:set_string("formspec", formspec.active(time))
 			else
 				meta:set_float("time", -1)
 				meta:set_string("output", "")
 				meta:set_string("formspec", formspec.inactive)
-				inventory:set_stack("main", 1, ItemStack(output))
+				local resultitem = ItemStack(output)
+
+				if output == "maidroid_core:custom" then
```

Comment:
hmm...
I don't want to embed the specialized code for the custom core into the core_writer.
I would rather create a custom core the same way as other cores, and create a new node that write a program to the custom core.